modules/kubernetes-addons/adot-collector-java/locals.tf

```diff
@@ -1,60 +0,0 @@
-locals {
-  name                           = "adot-collector-java"
-  adot_collector_service_account = "adot-collector-java"
-
-  default_helm_config = {
-    name        = local.name
-    repository  = null
```

Comment:
Maybe we could add a try in helm module [here](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/modules/kubernetes-addons/helm-addon/main.tf#L4)? that gives me an empty index issue

Reply:
Doing this `repository = try(var.helm_config["repository"], null)` in modules/kubernetes-addons/helm-addon/main.tf solved it for my tests